### PR TITLE
Added descriptions, enum members

### PR DESCRIPTION
**Pull request by irynaharoshka**
https://app.clickup.com/t/4535044/SP-7938
new enum members: FaceSDKResultCode, ImageSource

### CODE REVIEW

**Review by KirylKovaliov — APPROVED** [review]

File: detect.yml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-
 paths:
   /detect:
     parameters:
       - $ref: './common.yml#/components/parameters/x-request'
     post:
       tags:
         - matching
-      summary: "Detect face, evaluate attributes, assess the portrait quality"
+      summary: "detect faces"
```
Comment (ikliashchou) [inline detect.yml:9]: remove summary at all

File: common.yml [hunk 2]
```diff
@@ -449,7 +449,17 @@ components:
...
+      
+      Some of the values: 
+      
+      - `0`—everything is OK
+
+      - `1`—image is empty, try again with another image
+
+      - `2`—face is not detected
+
+      - `6`—data cannot be decoded as an image"
```
[the remark this answers is not in the document]
Comment (irynaharoshka) [reply common.yml:462]: removed in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files

File: detect.yml (hunk 1 above)
Comment (irynaharoshka) [reply detect.yml:9]: removed in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files

File: index.yml [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
   x-logo:
     url: 'https://static-content.regulaforensics.com/Icons/Logos/Regula-logo.svg'
     href: 'https://mobile.regulaforensics.com/'
   description: |
-    [Download OpenAPI specification](https://github.com/regulaforensics/FaceSDK-web-openapi)
+    <a href="https://regulaforensics.com/products/face-recognition-sdk/  " target="_blank">Regula Face SDK</a> is a cross-platform biometric verification solution for a digital identity verification process. The SDK enables convenient and reliable face capture on the client side (mobile, web, and desktop) and further processing on the client or server side.
+
+    The Face SDK includes the following features:
+    
+    * <a href="https://docs.regulaforensics.com/develop/face-sdk/overview/introduction/#face-detection" target="_blank">Face Detection</a>
```
[the remark this answers is not in the document]
Comment (irynaharoshka) [reply index.yml:13]: added in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files

File: match_and_search.yml [hunk 4]
```diff
@@ -85,6 +90,7 @@ components:
           properties:
             images:
               type: array
+              description: "Detected Persons."
```
[the remark this answers is not in the document]
Comment (irynaharoshka) [reply match_and_search.yml:93]: detected is ok here

File: matching.yml [hunk 5]
```diff
@@ -72,7 +82,7 @@ components:
                     type: integer
                 returnOriginalRect:
                   type: boolean
-                  description: "Whether to return the coordinates of the rectangle with the face in the original image prepared for the face crop."
+                  description: "Whether to return the coordinates of the rectangle with the face in the original image prepared for the face crop. Can be used only if the face in the original image is not tilted."
```
[the remark this answers is not in the document]
Comment (irynaharoshka) [reply matching.yml:85]: removed in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files

File: matching.yml [hunk 6]
```diff
@@ -82,7 +92,7 @@ components:
                     type: integer
                 type:
                   type: integer
-                  description: "The aspect ratio according to which face alignment is performed during face detection."
+                  description: "The aspect ratio according to which face alignment is performed. See the [FaceImageQualityAlignType enum](https://docs.regulaforensics.com/develop/face-sdk/web-service/development/enums/face-image-quality-align-type/). To get a thumbnail with dimensions 3x4, set `0`."
```
[the remark this answers is not in the document]
Comment (irynaharoshka) [reply matching.yml:95]: 'aspect ratio' is the most correct term, fixed in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files

**Pull request merged**